validator/config/config.py

```diff
@@ -0,0 +1,32 @@
+"""Nestor config validator configuration"""
+
+from enum import Enum
+import os
+
+
+class Configuration:
+    """Nestor_config_validator configuration"""
+
+    @staticmethod
+    def get_target_path():
+        """Returns the target path of the project configuration"""
+        return os.getenv("NESTOR_TARGET_PATH")
```

Comment:
It could be good to use `NESTOR_CONFIG_PATH` here has it probably serves the same purpose 🙂  (indicating where the directory holding the configuration is).